config/accounts_test.go

```diff
@@ -0,0 +1,114 @@
+package config
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestAccountGDPREnabledForRequestType(t *testing.T) {
+	tests := []struct {
+		description          string
+		giveRequestType      RequestType
+		giveGDPREnabled      *bool
+		giveAMPGDPREnabled   *bool
+		giveAppGDPREnabled   *bool
+		giveVideoGDPREnabled *bool
+		giveWebGDPREnabled   *bool
+		wantEnabled          *bool
+	}{
+		{
+			description:        "GDPR AMP integration enabled, general GDPR disabled",
+			giveRequestType:    RequestTypeAMP,
+			giveGDPREnabled:    &[]bool{false}[0],
+			giveAMPGDPREnabled: &[]bool{true}[0],
+			wantEnabled:        &[]bool{true}[0],
```

Comment:
what about this ?
```go
True, False := true, false
...
wantEnabled:        &True,
```

Reply:
.. or truePointer / falsePointer since it's weird for method scoped variables to start with a capital letter. Same idea.